zfit/models/functor.py

```diff
@@ -314,8 +314,8 @@ def __init__(self, pdfs: List[ZfitPDF], obs: ztyping.ObsTypeInput = None, name="
     def _unnormalized_pdf(self, x: ztyping.XType):
 
         norm_range = self._get_component_norm_range()
-        return ztf.reduce_prod([pdf.unnormalized_pdf(x, component_norm_range=norm_range.get_subspace(obs=pdf.obs))
-                                for pdf in self.pdfs], axis=0)
+        return np.prod([pdf.unnormalized_pdf(x, component_norm_range=norm_range.get_subspace(obs=pdf.obs))
```

Comment:
Interesting use of `numpy`. Why is `tensorflow` not good here? It's because we already have numbers?

Reply:
Yes, it's to use the overloaded `__mul__` operator. The probs are still tensors, but TensorFlow does not offer a "multiply n tensors", only the "reduce_prod", which requires though a fixed axes (but we wan't things to broadcast here)

So it's essentially just a * b * c...

Broadcast from (n, 1) * (1, m) -> (n, m) is needed

(tf.einsum would work as well)